src/sections/services.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div
+      class="font-[outfit] flex flex-col gap-24 lg:gap-0 lg:flex-row justify-around mt-12 bg-[#EFECEA] px-8 sm:px-28 lg:px-52"
+    >
```

Comment:
Let's configure our custom colors in the `tailwind.config.js`. 
```suggestion
    <div
      class="font-[outfit] flex flex-col gap-24 lg:gap-0 lg:flex-row justify-around mt-12 bg-primary px-8 sm:px-28 lg:px-52"
    >
```

Reply:
To add to this, you should set the background color on "body", since the whole page uses it.

Reply:
Since you have the font family set on the whole body element it's not needed to also add it here specifically.